app/components/markdown/markdown_code_block/markdown_code_block.js

```diff
@@ -165,9 +169,14 @@ export default class MarkdownCodeBlock extends React.PureComponent {
                     </View>
                     <View style={style.rightColumn}>
                         <View style={style.code}>
-                            <Text style={[style.codeText, this.props.textStyle]}>
+                            <SyntaxHighlighter
```

Comment:
Can you remove the syntax highlighting from this view on Android? While we can technically support it here, it'd better to be consistent with how we handle it across the the app for them.

Reply:
Sure. Will do it this weekend.